src/lib/containers/IconSvg.tsx

```diff
@@ -455,6 +462,16 @@ const getIcon = (options: IconSvgOptions) => {
       return <VisibilityTwoTone style={muiSvgStyle} />
     case 'visibilityOff':
       return <VisibilityOffTwoTone style={muiSvgStyle} />
+    case 'strikethrough':
+      return <StrikethroughS style={muiSvgStyle} />
+    case 'latex':
+      return <span>TeX</span>
+    case 'image':
+      return <Image style={muiSvgStyle} />
+    case 'superscript':
+      return <span>X&sup2;</span>
+    case 'subscript':
+      return <span>X&#8322;</span>
```

Comment:
Could not find appropriate icon for LaTeX, subscript, and superscript. There seems to be icons for subscript and superscript in MUI V5 which we can later update?

Reply:
What you have works!